### ISSUE TITLE
Can't find the NDI Library

### ISSUE THREAD

**Issue by fabiomb** [Palakis/obs-ndi#209]
My setup is almost complete, i have installed OBS, the obs-ndi plugin, the NDI redist v3 but still get this error:

11:36:38.974: [obs-browser]: Version 2.1.5
11:36:38.976: NVENC supported
11:36:38.978: [obs-ndi] hello ! (version 4.5.2)
11:36:38.978: [obs-ndi] Trying ''
11:36:38.978: [obs-ndi] Can't find the NDI library
11:36:40.495: Failed to initialize module 'obs-ndi.dll'
11:36:40.533: VLC found, VLC video source enabled

It's like the plugin can't find the files

OBS is in the C:\Program Files\obs-studio folder

NDI is in C:\Program Files\NewTek\NewTek NDI 3.5 Redist\v3 

any idea? i used the installer first and it worked fine, but the problem seems in the obs-ndi and where is looking for the files

**Comment by NoriegaMxl** [comment]
Which Version of OBS are you using?, I ve installed it in windowws and works fine... Also, What processor you have?...
Also: Only use this release with OBS 22 or above!

**Comment by fabiomb** [comment]
I'm using the last version of OBS, complete new install 22.0.2

**Comment by NoriegaMxl** [comment]
OS?... Processor?...

**Comment by fabiomb** [comment]
Core i7-6700 over Windows 10

**Comment by NoriegaMxl** [comment]
Did you tried with the 3.5 version?...

**Comment by fabiomb** [comment]
Yes i'm using the Redist 3.5+, that same file, i tried installing it
several times.

is there a problem with 64 bit OS? i think everyone here is using it in
64bits, do i need to register the dlls? never tried it on Win10

**Comment by fabiomb** [comment]
Now is working but without reason, just started without problem, no idea :|

**Comment by Palakis** [comment]
@fabiomb You need to reboot Windows after installing the NDI runtime.
